pkg/manifest/manifest_test.go

```diff
@@ -656,42 +680,42 @@ func Test_include(t *testing.T) {
 		},
 		{
 			name:        "default profile selection excludes without annotation",
-			profile:     DefaultClusterProfile,
+			profile:     &defaultClusterProfile,
 			annotations: map[string]interface{}{},
 			expected:    fmt.Errorf("include.release.openshift.io/self-managed-high-availability unset"),
 		},
 		{
 			name:        "default profile selection excludes with no annotation",
-			profile:     DefaultClusterProfile,
+			profile:     &defaultClusterProfile,
 			annotations: nil,
 			expected:    fmt.Errorf("no annotations"),
 		},
```

Comment:
add "no capability anno" test

Reply:
Now that capability is pointer i.e. `caps    *configv1.ClusterVersionCapabilitiesStatus` , when ever we not passing any capability it is passed as `nil`.

Reply:
Right, but I was referring to something more explicit but it's not needed.